main.py

```diff
@@ -1,52 +1,70 @@
 import uuid
 
 
-class Bank():
-    default_log='None'
-    def __init__(self,user_name,log=default_log):
-        self.user_name=user_name
-        self.log=str(log)
+class User():
+    def __init__(self,name):
+        self.name=name
         self.card=Card()
+
     def info(self):
-        print(f'User name:{self.user_name}')
-        return (self.card.info_card())
+        print(f'User name:{self.name}')
+        return (self.card.info())
+
+    def logg(self):
+        return(self.card.loggering())
 
 class Card:
     default_ballance=0
-    default_uuid=uuid.uuid4()
-    def __init__(self,uuid=default_uuid,ballance=default_ballance):
-        self.uuid=uuid
+    default_log=[]
+
+    def __init__(self,ballance=default_ballance,log=default_log):
         self.ballance=float(ballance)
+        self.log=log
+
+
+    def loggering(self):
```

Comment:
1. loggering -> logging. But logging - is process to add smth to log. 
2. What is idea behind this method?  - just print log info? then it's not logging... but smth like show_log, print_log or smth like this